src/model/encoding/convertFromRawToDraftState.js

```diff
@@ -62,7 +71,9 @@ function convertFromRawToDraftState(
     }
   );
 
-  return ContentState.createFromBlockArray(contentBlocks);
+  const contentState = ContentState.createFromBlockArray(contentBlocks);
```

Comment:
Might make sense for `createFromBlockArray` to accept a second parameter for the entity map, rather than requiring the `set` below.